proxy/src/main.rs

```diff
@@ -66,13 +106,16 @@ fn main() -> anyhow::Result<()> {
                 .help("redirect unauthenticated users to given uri")
                 .default_value("http://localhost:3000/psql_session/"),
         )
+        .arg(Arg::with_name("ssl-key"))
```

Comment:
These really need a help-text. That's all the documentation we have for any of these options at the moment.